api/envoy/api/v2/cds.proto

```diff
@@ -632,6 +632,30 @@ message Cluster {
   // If this flag is not set to true, Envoy will wait until the hosts fail active health
   // checking before removing it from the cluster.
   bool drain_connections_on_host_removal = 32;
+
+  // New mechanism for LB policy configuration. If set and understood
+  // by the client, replaces the lb_policy field, the common_lb_config
```

Comment:
Nit: please use `:ref:` links so that we get machine checked HREFs back to the field docs.